std_msgs/Header header
ConvexShape[] locations
